tests/appsec/test_asm_standalone.py

```diff
@@ -56,6 +59,40 @@ def _assert_tags_value(span, obj, expected_tags):
         except (KeyError, AssertionError) as e:
             return False
 
+    def setup_check_product_is_enabled(func):
+        def wrapper(self):
+            headers = {}
+            if self.tested_product is "appsec":
+                headers = {
+                    "User-Agent": "Arachni/v1",  # attack if APPSEC enabled
+                }
+            self.check_r = weblog.get(self.requestdownstreamUrl, headers=headers,)
+            func(self)
+
+        return wrapper
+
+    def check_product_is_enabled(func):
```

Comment:
I'm afraid that the usage of a decorator will be misleading for non python user. As this does not really decorate the test method, but just add a common assertion, don't you think that it will be more easier to understand if you name it `assert_product_is_enabled`, and call it directly in test methods with `assert_product_is_enabled(self.check_r, self.tested_product` ?

Reply:
I've changed it, thanks for the suggestion!